subgraph/package.json

```diff
@@ -14,7 +14,7 @@
     "generate:schema": "cp ./schemas/schema.${VERSION:-v1}.graphql schema.graphql",
     "prebuild": "pnpm codegen",
     "build": "graph build",
-    "deploy": "graph deploy --node https://api.studio.thegraph.com/deploy/ maci-subgraph",
+    "deploy": "graph deploy --node https://api.studio.thegraph.com/deploy/ test-maci-rpgf-subgraph",
```

Comment:
This might mean that the user should create subgraph on the studio website and named it `test-maci-rpgf-subgraph` right? maybe could put this description in the `README.md` file.

Reply:
I forgot to remove this name. User can use any unique name inside the subgraph studio account and this is described in studio dashboard.